pagefaultgames/pokerogue
# [Localization] Some minor modifications on Korean locales

## What are the changes?

I translated some names into Korean and revised some others to match the official name.

## Why am I doing these changes?



For correcting mistranslations and adding additional translations.

## What did change?


- In the current official version (Gen IX), the battle UI displays "싸운다" instead of "싸우다" and "도망친다" instead of "도망치다," so I made the corrections.
- "Oval Charm" was not translated, so I changed it to the official name.
- Translated "Toxic orb" and "Flame orb" to the official names and added descriptions in Korean.
- For "Mini Black Hole," the correct usage in Korean is "블랙홀" instead of "블랙 홀," so I made the revision.
- "BANETTITE" was incorrectly translated as "깜까미나이트" and has been corrected to "다크펫나이트."
- The word "egg" is officially translated as "알." Since .ts files except for "voucher.ts" file all follow the official name, "에그" in "voucher.ts" file has been changed to "알" for consistency.
- The word "ruins" can be translated to "폐허" or "유적" in Korean, but "유적" is more appropriate for ancient ruins. While "폐허" is not incorrect, it is often used to describe modern-day ruins caused by fire or war. Plus, the "Solaceon Ruins" from Generation 4 is officially translated as "신수유적."
- Among the biomes, Space was originally named Stratosphere and then changed to Space(as far as I know). In Korean, "stratosphere" is "성층권" and "space" is "우주," so I made the adjustment.
- The Fairy type is officially translated as "페어리 타입" in Korean. Therefore, "Fairy Cave" should be "페어리 동굴" instead of "요정 동굴" to show its connection with the Fairy Type.

### Screenshots/Videos


![a1](https://github.com/pagefaultgames/pokerogue/assets/171243264/6f9fb021-8f2e-411d-9740-c857cc7139e2)
![a2](https://github.com/pagefaultgames/pokerogue/assets/171243264/cf5811f2-9b2a-4528-b126-9477b24381e9)
![a3](https://github.com/pagefaultgames/pokerogue/assets/171243264/259e8d37-6e73-4c7e-9f51-d027e4d7ba88)
![a4](https://github.com/pagefaultgames/pokerogue/assets/171243264/53e314ee-b029-4554-8e2c-e0e697c03a9d)

## How to test the changes?




- Start the game and enter a battle to check the UI.
- Verify changes by modifying items or locations in the override.ts file.
- Confirm the name changes in the voucher menu.

## Checklist
